### DISCUSSION TITLE
Get a page from a collection by ID

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#7770] [General]
I'm passing params to a thank you page that displays a custom message. I've got the id but I need to find the id and display the page message contents in my antlers view.

```
  {{ entries collection="landing_pages" id="ff34d1c0-3c24-4564-80cb-d529ff700706" }}
               {{ message_title }}
               {{ message_desc }}
  {{ /entries }}
```

**Comment by duncanmcclean** [comment]
Try using the get_content tag instead.
